skills/sweep-issues/SKILL.md

```
@@ -74,6 +76,41 @@ If a prior rejection covers the same idea, **skip the candidate silently** and n
+if [ -n "$branch" ] && [ "$branch" != "main" ] && [ "$branch" != "master" ]; then
+  changed_files=$(git diff origin/main --name-only 2>/dev/null)
+fi
+```
+
+If `$changed_files` is non-empty, for every candidate categorized as `bug`:
+
+1. Match the candidate against the diff. A candidate is **PR-touched** if any of:
+   - Its description references a file path in `$changed_files` (literal or basename match)
+   - Its description references a symbol (function, type, constant) that appears in `git diff origin/main` as an added or modified line
+2. If PR-touched, use AskUserQuestion individually:
+   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)